src/components/Footer.astro

```diff
@@ -0,0 +1,25 @@
+---
+import {SOCIAL_LINKS} from "../constants";
+---
+<footer class="footer p-10 bg-base-300 text-base-content mt-20">
```

Comment:
Need to review with @santoshyadavdev about the links we want to use in the footer